src/services/objection.service.ts

```diff
@@ -21,10 +21,10 @@ import {
  * @returns {string} the id of the newly created objection
  * @throws {ApiError}
  */
-export const createNewObjection = async (companyNumber: string, token: string): Promise<string> => {
+export const createNewObjection = async (companyNumber: string, token: string, create: ObjectionCreate): Promise<string> => {
```

Comment:
ide has highlighted line 24
Parameter create is not described in JSDoc

Reply:
Updated the doc